pkg/antctl/raw/multicluster/leave.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+	"fmt"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+type leaveOptions struct {
+	namespace  string
+	clusterSet string
+}
```

Comment:
This option is just the same as `destroyOptions`, you may rename and reuse it.
Please wrap some methods to reduce code duplicate as we discussed.

Reply:
move all the common code th `common.clean_resource.go`